### DISCUSSION TITLE
mpv formula does not symlink .pc file to (brew --prefix)/lib/pkgconfig

### DISCUSSION THREAD

**Discussion by guillaumekh** [Homebrew/discussions#3468] [Writing Formulae/Casks] [answered]
Hi,

I have noticed that the [mpv](https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/mpv.rb) formula does not symlink its `mpv.pc` pkgconfig file to `(brew --prefix)/lib/pkgconfig/` like many other formulas do.

I'm willing to submit a PR to get the formula to do this but I'm failing to understand what controls this behaviour, despite pouring over formulas which generate a symlink (e.g. zlib).

Can someone point me in the right direction?

Kind regards,
Guillaume

**Comment by carlocab** [comment, marked as answer]
> I have noticed that the [mpv](https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/mpv.rb) formula does not symlink its `mpv.pc` pkgconfig file to `(brew --prefix)/lib/pkgconfig/` like many other formulas do.

This is not true.
```
❯ find /usr/local/lib/pkgconfig -name 'mpv*' -print
/usr/local/lib/pkgconfig/mpv.pc
❯ file /usr/local/lib/pkgconfig/mpv.pc
/usr/local/lib/pkgconfig/mpv.pc: symbolic link to ../../Cellar/mpv/0.34.1_1/lib/pkgconfig/mpv.pc
```
The issue is probably something on your system. Have you tried
```
brew link mpv
```
?

**Reply by guillaumekh** [reply]
Damn me you're right. `brew unlink` & `brew link` did the trick.
Please accept my apologies for not doing my homework properly first !
Thank you for the help.
